internal/controller/api/v1alpha1/image_types.go

```diff
@@ -17,25 +17,28 @@ limitations under the License.
 package v1alpha1
 
 import (
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
 // ImageSpec defines the desired state of Image
 type ImageSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
-	// Foo is an example field of Image. Edit Image_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// The name of the Cluster CR
+	ClusterName  corev1.ObjectReference `json:"clusterName"`
```

Comment:
Should be ClusterRef according to the naming conventions:

> The name of a field referring to another resource of kind Foo by name should be called fooName. The name of a field referring to another resource of kind Foo by ObjectReference (or subset thereof) should be called fooRef.

Reply:
Done